readme.md

```diff
@@ -755,9 +755,9 @@ Fugiat non in itaque sunt nobis totam. Sed nesciunt est deleniti cumque alias. R
 <?php
 
 echo $faker->idNumber;      // ID number
-echo $faker->nationalIdNumber // Citizen ID number
-echo $faker->foreignerIdNumber // Foreigner ID number
-echo $faker->companyIdNumber // Company ID number
+echo $faker->nationalIdNumber; // Citizen ID number
+echo $faker->foreignerIdNumber; // Foreigner ID number
+echo $faker->companyIdNumber; // Company ID number
```

Comment:
How about line 768? Did you check all the examples?

Reply:
Oops, I missed 768, I added that one now and a few others too, I went through the whole readme file again, should be all now.